code/game/objects/items/reagent_containers/autoinjectors.dm

```diff
@@ -113,18 +113,33 @@
 /obj/item/reagent_containers/hypospray/autoinjector/hyperzine/expired
 	name = "expired hyperzine autoinjector"
 	desc = "An auto-injector said to be loaded with a safe-to-use hyperzine mix, 3 months past it's expiration date."
-	list_reagents = list(/datum/reagent/medicine/hyperzine = 4,
-						/datum/reagent/mercury = 1,
-						/datum/reagent/medicine/dexalin = 8,
-						/datum/reagent/medicine/inaprovaline = 8,
-						/datum/reagent/toxin = 4)
+	list_reagents = list(/datum/reagent/medicine/hyperzine = 4, /datum/reagent/medicine/dexalin = 8, /datum/reagent/medicine/inaprovaline = 8, /datum/reagent/toxin = 5)
```

Comment:
Rather keep the old format, it was easier to read.

Reply:
but it doesn't line up right aaaAAaaAA

fine